test/Swagger.ObjectModel.Tests/Builders/ExternalDocumentationBuilderTest.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Swagger.ObjectModel.Builders;
+using Xunit;
+
+namespace Swagger.ObjectModel.Tests.Builders
+{
+    public class ExternalDocumentationBuilderTest
+    {
+        private readonly ExternalDocumentationBuilder builder;
+        private readonly string url = string.Empty;
+        public ExternalDocumentationBuilderTest()
+        {
+            this.builder = new ExternalDocumentationBuilder();
+            this.url = "https://github.com/yahehe/Nancy.Swagger";
+        }
+
+        [Fact]
+        public void Should_ThrowRequiredFieldException_WhenUrlIsNullOrEmpty()
```

Comment:
Oh really? I'm actually usually a fan of using _ to separate sentences like that:
[Should|ShouldNot]_[DoThisThing]_[When|If|ThisThing]

Reply:
I just feel like the keywords are helpful enough. 

I'm fine either way though - I mostly brought it up because there was the precedent not to use them. Maybe rename the existing functions then?

Reply:
I find some Open Source projectss use this style for their tests , such as NancyFx . I use _ to separate for both of my own projects as well , so what I did is just follow this style .